python/setup-css.py

```diff
@@ -27,12 +27,16 @@ def run_tests(self):
         errno = pytest.main(self.pytest_args)
         sys.exit(errno)
 
+    with open("README.md", "r", encoding="utf-8") as fh:
+        long_description = fh.read()
+
 
 setup(
     name="cssbeautifier",
     version=__version__,
     description="CSS unobfuscator and beautifier.",
-    long_description=("Beautify, unpack or deobfuscate CSS"),
+    long_description=long_description,
```

Comment:
Please revert lines 30-32 and then
```suggestion
    long_description=Path("README.md").read_text(),
```
and then at the top of the file add `from pathlib import Path`

Reply:
Ok thanks